src/inference/include/openvino/runtime/remote_context.hpp

```diff
@@ -106,6 +106,12 @@ class OPENVINO_RUNTIME_API RemoteContext {
      */
     ~RemoteContext();
 
+    /**
+     * @brief method: checks remote is empty or not
+     * @return The context is empty return true, else false.
+     */
+    bool is_empty() const;
+
```

Comment:
let's maybe create ov::RemoteTensor::operator bool() const in a separate PR?
and reuse it also in core_impl.cpp instead of check for `context._impl` ?

Reply:
Will create a PR for this

Reply:
@ilya-lavrenov  PR created, pls review https://github.com/openvinotoolkit/openvino/pull/18186